frontend/store/auth-context.js

```diff
@@ -10,16 +11,31 @@ const AuthContext = React.createContext({
 
 export const AuthContextProvider = (props) => {
   const [token, setToken] = useState("");
-  const isLoggedIn = token.length !== 0;
+  const isLoggedIn = token.length !== 0
 
   const loginHandler = (token) => {
     setToken(token);
+    AsyncStorage.setItem('token',token);
   };
 
   const logoutHandler = () => {
     setToken("");
+    AsyncStorage.setItem('token',"");
   };
 
+  const isLogIn = async () => {
+      try {
+        let savedToken = await AsyncStorage.getItem('token');
+        if (savedToken) setToken(await userToken);
```

Comment:
```suggestion
        if (savedToken) setToken(savedToken);
```

Reply:
Alright! now it functions correctly!
Thanks for the fix!